ansible/playbook-setup-quince.yml

```diff
@@ -0,0 +1,49 @@
+- hosts: quince.gentoo-ev.org
+  gather_facts: no
+
+  tasks:
+
+  - name: "Stop and mask service rpcbind (to no longer expose TCP port 0.0.0.0:111)"
+    systemd:
+      name: "{{ item }}"
+      enabled: no
+      masked: yes
+      state: stopped
+    with_items:
+    - rpcbind.service
+    - rpcbind.socket
+
+  - name: Add yum repository "epel" (for htop and mg)
+    yum_repository:
+      name: epel
+      description: Extra Packages for Enterprise Linux $releasever - $basearch
+      metalink: https://mirrors.fedoraproject.org/metalink?repo=epel-$releasever&arch=$basearch&infra=$infra&content=$contentdir
+      enabled: yes
+      gpgcheck: yes
+      gpgkey: https://archive.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-$releasever
+
+  - name: Install distro packages
+    package:
+      state: present
+      name:
+    # Debugging tools
+      - htop
+      - lsof
+      - mg
+      - nano
+      - tmux
+      - wget
+
+  - name: Add users
+    user:
+      name: "{{ item }}"
+    with_items:
+      - backup
+
+  - name: Create backup directory
+    file:
+      path: /home/backup/banana
+      state: directory
+      owner: backup
+      group: backup
+      mode: 0755
```

Comment:
Maybe not the most important, but should mode be 0755 in theory? Maybe 0700 instead?

Reply:
Good catch. I've made it 0750 because it is in its own group.